test/controllers/pages_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require "test_helper"
+
+class PagesControllerTest < ActionDispatch::IntegrationTest
+  test "dashboard" do
+    sign_in users(:bob)
```

Comment:
Can we move the sign in do a `setup do` block?  I think that should be a consistent pattern throughout regardless of how small the test file is (as it may grow in the future)

Reply:
Fair call, done.